azure-pipelines.yml

```diff
@@ -30,11 +31,13 @@ steps:
     $DOCKER_OVERRIDE up --no-build -d
     $DOCKER_OVERRIDE exec -T web /bin/sh -c "./wait-for-command.sh -c 'nc -z db 5432' -s 0 -t 20"
     $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rails db:setup"
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rake cc:setup"
```

Comment:
can we move the curl install to here as its not really needed for the image

Reply:
Good shout

Reply:
Moved `apk add curl` out of docker image and into the pipeline as suggested by Sau

Reply:
beautiful